compute_tools/src/compute.rs

```diff
@@ -51,11 +51,17 @@ use crate::{config, extension_server, local_proxy};
 
 pub static SYNC_SAFEKEEPERS_PID: AtomicU32 = AtomicU32::new(0);
 pub static PG_PID: AtomicU32 = AtomicU32::new(0);
+// This is an arbitrary build tag. Fine as a default / for testing purposes
+// in-case of not-set environment var
+const BUILD_TAG_DEFAULT: &str = "latest";
```

Comment:
You could probably just inline `latest` in `Lazy::new()` since I have a suspicion that is the only time that this variable is referenced. Up to you